pkg/opni/commands/import_progress.go

```diff
@@ -38,12 +38,8 @@ func getNextStatus(ctx context.Context, request *remoteread.TargetStatusRequest)
 }
 
 func getProgressAsPercent(progress *remoteread.TargetProgress) float64 {
-	if progress == nil || progress.LastReadTimestamp == nil {
-		return 0
-	}
-
-	percent := float64(progress.LastReadTimestamp.Seconds-progress.StartTimestamp.Seconds) /
-		float64(progress.EndTimestamp.Seconds-progress.StartTimestamp.Seconds)
+	percent := float64(progress.GetLastReadTimestamp().GetSeconds()-progress.GetStartTimestamp().GetSeconds()) /
+		float64(progress.GetEndTimestamp().GetSeconds()-progress.GetStartTimestamp().GetSeconds())
```

Comment:
Need a check to prevent dividing by zero here

Reply:
whoops thanks